attestationreport/validationreport.go

```diff
@@ -166,10 +165,11 @@ type SignatureResult struct {
 	ValidatedCerts  [][]x509CertExtracted `json:"validatedCerts"`        //Stripped information from validated x509 cert chain(s) for additional checks from the policies module
 	SignCheck       Result                `json:"signatureVerification"` // Result from checking the signature has been calculated with this certificate
 	CertChainCheck  Result                `json:"certChainValidation"`   // Result from validatint the certification chain back to a shared root of trust
-	RoleCheck       *Result               `json:"roleCheck,omitempty"`   // Result for checking the role in the certificate (optional)
 	ExtensionsCheck *ResultMulti          `json:"extensionsCheck,omitempty"`
 }
 
+//X509CertExtracted represents a x509 certificate
+// with attributes in a human-readable way and prepared for (Un)Marshaling JSON objects
```

Comment:
Consistent space between Comment // and Text and exact name: 
// x509CertExtracted

Though which should rename the struct to X509CertExtracted

Reply:
done